src/Workspaces/Core/Portable/FlowAnalysis/SymbolUsageAnalysis/SymbolUsageAnalysis.Walker.cs

```diff
@@ -309,7 +309,7 @@ private void AnalyzeLocalFunctionInvocation(IMethodSymbol localFunction)
             {
                 Debug.Assert(localFunction.IsLocalFunction());
 
-                var newAnalysisData = _currentAnalysisData.AnalyzeLocalFunctionInvocation(localFunction, _cancellationToken);
+                var newAnalysisData = _currentAnalysisData.AnalyzeLocalFunctionInvocation(localFunction.OriginalDefinition, _cancellationToken);
```

Comment:
This is the only real product change.

Reply:
interesting.  should this potentially happen *under* the covers? i.e. should AnalyzeLocalFunctionInvocation just do this?  or is there merit in analyzin the instantiated vs non-instantiated local functions?

Reply:
Yes, I can also move it down to `AnalyzeLocalFunctionInvocationCore` instead of having the asserts in the callee. Let me simplify it.